parser/html_to_dita.py

```diff
@@ -167,23 +167,45 @@ def htmlToDITA(file_name, soup_in, dita_soup):
 
     # 9. For top-level block-quotes that contain `p` elements, switch to UL lists
     for bq in soup.find_all("blockquote", recursive=False):
-        if bq.find("blockquote"):
-            # blockquotes used for padding. replace with placeholder
-            para = dita_soup.new_tag("p")
-            para.string = "[WHITESPACE FOR TABLE]"
-            bq.replace_with(para)
-        else:
-            # note: we aren't doing this recursively, we're just looking at the top level
-            # in other scenarios there are nested blockquotes, culminating in a heading in a p
-            # we need to handle them separately.
-            if bq.find("p", recursive=False):
-                # it's an indented list.
-                bq.name = "ul"
-                for p in bq.find_all("p", recursive=False):
-                    p.name = "li"
+            if bq.find("blockquote"):
+                # blockquotes used for padding. replace with placeholder
+                para = dita_soup.new_tag("p")
+                para.string = "[WHITESPACE FOR TABLE]"
+                bq.replace_with(para)
+            else:
+                # note: we aren't doing this recursively, we're just looking at the top level
+                # in other scenarios there are nested blockquotes, culminating in a heading in a p
+                # we need to handle them separately.
+                if bq.find("p", recursive=False):
+                    # it's an indented list.
+                    bq.name = "ul"
+                    for p in bq.find_all("p", recursive=False):
+                        p.name = "li"
+
+        # for ul in soup.find_all('ul'):
+        #     ul.name = 'ol'
+
+    for td in soup.find_all("td"):
+        td.name = "entry"
+        td['colspan'] = "1"
+
+    # Replace `span` used for red-formatting with a <ph> equivalent
+    for span in soup.find_all("span"):
+        if span.has_attr("style"):
+            if "color:#F00" in span["style"]:
+                span.name = "ph"
+                span["outputclass"] = "red"
+                del span["style"]
 
     # for ul in soup.find_all('ul'):
     #     ul.name = 'ol'
+    for td in soup.find_all("td"):
```

Comment:
Yes, but in the `htmlToDita` function i already have a block to change the name of the `td` to `entry`
```js
   if soup.name == "td":
        soup.name = "entry"
        soup['colspan'] = "2"
```

but this code is not changing some of the `td` elements to `entry` and the reason is because there are nested `td` elements like this 

```html
 <td> 
   <td> x </td>
  <td> y </td>
</td>
```

and the `if soup.name == "td"` code is not chaning the inner `td` elements, that is the reason i added the `for loop` to the `htmlToDita` function